lib/tasks/link_checker.rake

```diff
@@ -21,6 +21,15 @@ task :link_checker, [:date] => [:environment] do |_, _args|
   links = Dir.glob("config/locales/*.yml").map { |locale_filename| gather_urls YAML.safe_load(File.read(locale_filename)) }
   links.flatten!.uniq!
 
+  gov_links = []
+  other_links = []
+  links.flatten.uniq.each do |uri|
+    if URI(uri).host.ends_with? ".gov.uk"
+      gov_links << uri
```

Comment:
Why are these links being separated?

Reply:
the gov_urls and other_urls are now checked in different ways.
other_links are checked for access and speed
gov_links are checked for unpublished/published.

Reply:
I think that happens at a later point in the history, so when I was reviewing this commit by commit, it didn't make sense.